README.md

```diff
@@ -48,14 +48,65 @@ Setup development environment
 
 Possible fields in config file:
-
+- `authUrl` - URL to Keystone
+- `gohan` - Gohan connection config, must be an object with following fields:
+  - `schema` - path to JSON Schema
+  - `url` - URL to Gohan server
+- `sidebar` - array of custom sidebar buttons, must contain objects with following fields:
+  - `path`
+  - `title` 
+  - `id`
+- `sidebarFavorites` - array of links to views, must contain resources' IDs
+- `sidebarCategories` - array of objects including information about views' categories, must contain objects with following fields:
+  - `name` - display name of category
+  - `id` - ID of category
+  - `items` - array of links to views in the category
 - `sidebarChildResources` - array of schema IDs of child resources which should be visible in sidebar menu
 - `routes` - config array for routing, must contain objects with following fields:
   - `path` - URL path to view
   - `viewClass` - name of React component related to the view (in `src/routes/componentList.js`)
   - `name` - name of route
   - `exact` - match path exactly, default `true`
+- `errorMessages` - map of error messages
+  - `tokenExpire` - token expired message
+- `addingRelationDialog` - array of ????
+- `pageLimit` - limit of displayed items in page
+- `loginRequestTimeout` - time left (in ms) to token expiration (when token renew dialog should be shown)
+- `polling` - flag to enable polling (must be true or false)
+- `pollingInterval` - interval (in ms) of polling
+- `storagePrefix` - prefix to store token information (local- and sessionStorage)
+- `tableDefaultSortKey` - default sort key
+- `tableDefaultSortOrder` - default sort order (asc or desc)
+- `title`- title of web page
+- `followableRelations` - flag to enable following resource by reference
+- `useKeystoneDomain` - flag to enable use domain feature in keystone v3
+- `selectDomainFromHost` - regex to select domain from host (keystone v3)
+- `domainName` - default domain name
+
+
+UI schema
+----------------------------------------------------------------------------------------------------------------------
+`locales/en-us/uiSchema.json` file containg the definition of specific UI behaviors in dialogs like hiding and showing fields
+
+The JSON file should contain an array of objects with following fields:
+ - `id` - ID of JSON Schema resource
+ - `title` - (optional) - title of resource
+ - `properties` - (optional) - properties customization, objects with following fields:
+    - `[fieldName]` - (optional) - customization of single field, objects with following fields:
+       - `ui:widget` - (optional) -  alternative widget
+       - `ui:field` - (optional) - alternative field
+       - `ui:step` - (optional) - specified step in number input field
+       - `ui:labelTemplate` - (optional) - label custom template for select field
+       - `ui:requiredResource` - (optional) - custom require collection for select field
+       - `ui:query` - (optional) - query to fetch additional data for select field
```

Comment:
I'm not sure if that's all the fields we use to customize properties.
Please check `ui:order`, `ui:step`.

Reply:
`ui:step` is described on 102 line,
`ui:order` is described in react jsonschema form documentation